### PR TITLE
Ws 0.2.7 update fetch sighting approve

### PR DESCRIPTION
**Pull request by PrasantiNalla**
image

### CODE REVIEW

File: WhaleSpotting/Controllers/WhaleSightingController.cs [hunk 1]
```diff
@@ -27,4 +31,28 @@ public IActionResult GetById([FromRoute] int Id)
             return NotFound();
         }
     }
+
+    [HttpPatch("{id}/Approve")]
```
Comment (francescastocco) [inline WhaleSpotting/Controllers/WhaleSightingController.cs:35]: Can we make this lowercase and just for consistency swap the order?
```suggestion
    [HttpPatch("approve/{id}")]
```

File: WhaleSpotting/Services/LoginService.cs [hunk 2]
```diff
@@ -28,4 +29,19 @@ public bool IsValidLogin(string username, string password)
...
+            var user = _users.GetByUsername(username);
+            return user.UserType==(UserType) 1;
+        }
+        catch (Exception)
+        {
+            return false;
+        }
+    }
+
+
```
Comment (francescastocco) [inline WhaleSpotting/Services/LoginService.cs:46]: Extra line here :(

**Review by francescastocco — CHANGES_REQUESTED** [review]
This looks really good :)

File: WhaleSpotting/Controllers/WhaleSightingController.cs (hunk 1 above)
Comment (PrasantiNalla) [reply WhaleSpotting/Controllers/WhaleSightingController.cs:35]: changed

File: WhaleSpotting/Services/LoginService.cs (hunk 2 above)
Comment (PrasantiNalla) [reply WhaleSpotting/Services/LoginService.cs:46]: removed

**Review by francescastocco — APPROVED** [review]
Nice!

**Pull request merged**